wp-content/themes/core/blocks/sectionnav/sectionnav.php

```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
+
+/**
+ * @var array $args Arguments passed to the template
+ */
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$model = new \Tribe\Project\Blocks\Types\Section_Nav\Section_Nav_Model( $args['block'] );
+get_template_part( 'components/blocks/section_nav/section_nav', null, $model->get_data() );
```

Comment:
Since this is literally a "block" placeholder for the section nav component, do we need this? Further down all we're doing is farther down is passing this same data to the section_nav component. Seems like an unnecessary shell.

Reply:
Hmmm... I suppose we _could_. I'm trying to consider that in the future a design or scope decision may require adding additional fields/settings/values to the _block_ which could then be different than the base component. But, that may be overkill, too. I do like the separation between the content-entered "block" and the generic template part "component" though. Lemme stew on this over the weekend.

Reply:
👍  I don't consider this a blocking issue as it does have some utility as mentioned.